paf-mvp-frontend/tests/test-cases/onekey-lib.test.ts

```diff
@@ -418,7 +435,85 @@ describe('Function createSeed', () => {
     });
   });
 });
-
+describe('Function registerTransmissionResponse', () => {
```

Comment:
If not too difficult, I suggest that you put those tests in another file.
Like `onekey-lib.register-transmission.test.ts`. Up to you if you want to improve the name. :-)